content/docs/example.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Example
```

Comment:
Why this `example.md` page? As a sandbox for writers?

Reply:
No, it's more of a sandbox for experimenting with design elements. It's a draft so it isn't displayed in the production site.